R/bernoulli.R

```diff
@@ -157,3 +157,17 @@ cdf.bernoulli <- function(d, x, ...) {
 quantile.bernoulli <- function(d, p, ...) {
   qbinom(p = p, size = 1, prob = d$p)
 }
+
+#' Fit a bernoulli distribution to data
+#'
+#' @param d a `bernoulli` object
+#' @param x a vector of zeroes and ones to fit the bernoulli distribution to
```

Comment:
I like to use full sentences with correct punctuation in documentation, if you don't mind.

Reply:
I've updated the docs for each fit method to use proper punctuation